### ISSUE TITLE
Base class for Regression

### ISSUE THREAD

**Issue by sankhaMukherjee** [sankhaMukherjee/myCutter#15]
Generate a base class for a regressor that can be used for generating a baseline regression class.

This should include the following:

[ ] Allow a simple SkLearn regressor that will train on a problem
[ ] Allow a simple Tensorflow model to train on a set of problems
[ ] Allow a simple PyTorch model to train on a set of problems
[ ] A set of accuracy scores, and different types of visualization that is used for quantifying the classification algorithm
